src/components/prescriptions/PrescriptionComponent.vue

```diff
@@ -119,4 +119,79 @@
             @include text-header6($font-weight-regular);
         }
     }
+
+    @media (min-width: $zero-width) and (max-width: $tablet-width) {
+        .documentBox {
+            @include flex-position(row, nowrap, space-between, center);
+            width: 100%;
+            border: $border-size solid $gray-300;
+            border-radius: $border-radius--normal;
+            padding: 0.5rem;
+            background-color: $white;
+            gap: 2.5rem;
+            &:deep(svg) {
+                color: $blue-700;
+                height: 2rem;
+            }
+            &__title {
+                @include text-header6--tablets($font-weight-medium);
+                color: $gray-900;
+                text-align: center;
+            }
+            &__data {
+                @include text-header6--tablets($font-weight-medium);
+                text-align: center;
+            }
+        }
+    }
+
+    @media (min-width: $tablet-above-width) and (max-width: $laptop-width) {
+        .documentBox {
+            @include flex-position(row, nowrap, space-between, center);
+            width: 100%;
+            border: $border-size solid $gray-300;
+            border-radius: $border-radius--normal;
+            padding: 1rem;
+            background-color: $white;
+            gap: 3.75rem;
+            &:deep(svg) {
+                color: $blue-700;
+                height: 2.5rem;
+            }
+            &__title {
+                @include text-header6--laptop($font-weight-medium);
+                color: $gray-900;
+                text-align: center;
+            }
+            &__data {
+                @include text-header5--laptops($font-weight-medium);
+                text-align: center;
+            }
+        }
+    }
+
+    @media (min-width: $laptop-above-width) and (max-width: $pc-width) {
```

Comment:
same here. maybe declare only low border of range without closing

Reply:
jak wyzej